src/main.ts

```diff
@@ -1,72 +1,417 @@
+
+/**
+ * Raw Protobuf interfaces for a MovementSensor component.
+ *
+ * Generated with https://github.com/improbable-eng/grpc-web
+ *
+ * @deprecated Use {@link MovementSensorClient} instead.
+ * @alpha
+ * @group Raw Protobufs
+ */
+export { default as movementSensorApi } from './gen/component/movementsensor/v1/movementsensor_pb';
 export {
   type MovementSensor,
   MovementSensorClient,
 } from './components/movementsensor';
+
+/**
+ * Raw Protobuf interfaces generated with
+ * https://github.com/improbable-eng/grpc-web for a Sensor component.
+ *
+ * @deprecated Use {@link SensorClient} instead.
+ * @alpha
+ * @group Raw Protobufs
+ */
+export { default as sensorApi } from './gen/component/sensor/v1/sensor_pb';
 export { type Sensor, SensorClient } from './components/sensor';
+
+/**
+ * Raw Protobuf interfaces for a Sensors service.
+ *
+ * Generated with https://github.com/improbable-eng/grpc-web
+ *
+ * @example
+ *
+ * ```ts
+ * import { grpc } from '@improbable-eng/grpc-web';
+ *
+ * const client = {}; // replace with a connected robot client
+ *
+ * const request = new sensorsApi.GetSensorsRequest();
+ * request.setName('my sensors');
+ *
+ * client.sensorsService.getSensors(
+ *   request,
+ *   new grpc.Metadata(),
+ *   (error, response) => {
+ *     // do something with error or response
+ *   }
+ * );
+ * ```
+ *
+ * @alpha
+ * @group Raw Protobufs
+ */
+export { default as sensorsApi } from './gen/service/sensors/v1/sensors_pb';
+
+/**
+ * Raw Protobuf interfaces for a Stream.
+ *
+ * Generated with https://github.com/improbable-eng/grpc-web
+ *
+ * @deprecated Use {@link StreamClient} instead.
+ * @group Raw Protobufs
+ */
+export { default as streamApi } from './gen/proto/stream/v1/stream_pb';
 export { type Stream, StreamClient } from './extra/stream';
 
-export {
-  commonApi,
-  armApi,
-  baseApi,
-  boardApi,
-  cameraApi,
-  gantryApi,
-  genericApi,
-  gripperApi,
-  inputControllerApi,
-  motorApi,
-  motionApi,
-  movementSensorApi,
-  servoApi,
-  navigationApi,
-  robotApi,
-  sensorApi,
-  sensorsApi,
-  slamApi,
-  visionApi,
-  streamApi,
-  Client,
-  version,
-};
-
-// It doesn't matter which ServiceError we export really.
+/**
+ * Raw Protobuf interfaces for a Gantry component.
+ *
+ * Generated with https://github.com/improbable-eng/grpc-web
+ *
+ * @example
+ *
+ * ```ts
+ * import { grpc } from '@improbable-eng/grpc-web';
+ *
+ * const client = {}; // replace with a connected robot client
+ *
+ * const request = new gantryApi.GetLengthsRequest();
+ * request.setName('my gantry');
+ *
+ * client.gantryService.getLengths(
+ *   request,
+ *   new grpc.Metadata(),
+ *   (error, response) => {
+ *     // do something with error or response
+ *   }
+ * );
+ * ```
+ *
+ * @alpha
+ * @group Raw Protobufs
+ */
+export { default as gantryApi } from './gen/component/gantry/v1/gantry_pb';
+
+/**
+ * Raw Protobuf interfaces for a Generic component.
+ *
+ * Generated with https://github.com/improbable-eng/grpc-web
```

Comment:
I'm going to leave this one without an example until someone asks for it.

Reply:
I think Generic is a good one to have an example for actually